src/BxHelpLink.jsx

```diff
@@ -6,7 +6,7 @@ class BxHelpLink extends React.Component {
 
     return (
       <a
-        href={`https://solana-labs.github.io/book/terminology.html#${term}`}
+        href={`https://solana-labs.github.io/book-edge/terminology.html#${term}`}
```

Comment:
Let's select "book" or "book-edge" based on the value of testnetDefaultChannel [1].  If testnetDefaultChannel `===` "edge", use "book-edge", else "book"


[1] https://github.com/solana-labs/solana-web3.js/blob/master/package.json#L27

Reply:
@mvines ok, will mock something up. I'm a little fuzzy on how that will change/be overridden though (is it changed on deployment?)... isn't package.json immutable? In any event, patch coming...